src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -242,6 +242,9 @@ protected function loadConfig(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln('<info>using config parser</info> ' . $parser);
+
+        $output->writeln('<info>using migration base class</info> ' . $config->getMigrationBaseClassName(false));
```

Comment:
I personally would also be showing the template being used. With consideration of my other PR regarding alternative templates and template generating class, maybe moving them all to the ``create`` command would be a sensible place. Only showing the actual use (i.e. if overriding the template, then don't show the base template name). Happy with whatever you feel is appropriate. I just need the functionality. :-D